docker-compose.yml

```diff
@@ -114,24 +113,6 @@ services:
     ports:
       - "0.0.0.0:6379:6379"
 
-  worker:
```

Comment:
why did this get moved to another file? In `nox` we should always be specifying which service(s) to bring up:

👍 `docker compose up fides`
👎 `docker compose up`

so having it in this file seems ok?

Reply:
This is me following an old `ops` convention, we have a bunch of `docker-compose.integration-{...}.yml` files, due to the philosophy that `docker compose up` should bring up the minimum amount of infra possible to run the system on. Happy to add this back in if there's a new convention in Fides 2.0.

Reply:
not a new convention really, but in my opinion its a much simpler experience to do `docker compose up fides worker` than `docker compose -f docker-compose.yml -f docker-compose.worker.yml up`. Specifying which service to bring up feels better and more deliberate to me, do you have strong opinions otherwise?